ios/CollectorManager.swift

```diff
@@ -15,6 +15,30 @@ class CollectorManager: NSObject {
         
     CollectorManager.map[name] = VGSCollect(id: vaultId, environment: environment);
  }
+  
+    @objc(pinConfirm:path:method:headers:resolver:rejecter:)
+    func pinConfirm(_ name: String, path: String, method: String, headers: Dictionary<String, String>, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Bool {
+      let collector = CollectorManager.map[name]!
+        
+      if (collector == nil) {
+          reject("error", "invalid name", NSError(domain: "MissingCollectorByNameError", code: 401))
+          return;
+      }
+      collector.customHeaders = headers;
```

Comment:
I don't think we will be using headers for pin confitmation.

Reply:
Thanks, i removed the parameters that we don't use [here](https://github.com/Instawork/vgs-collect-react-native/pull/1/commits/65850574faec0cae00f0d2b793cf1529b5161f2f#diff-a41cc746377eae3513fca397551becca1eaaf1350440859511e60f1cb4dec243L34)